warp10/src/main/java/io/warp10/continuum/Tokens.java

```diff
@@ -253,7 +253,28 @@ public static ReadToken extractReadToken(String token) throws WarpScriptExceptio
     
     List<String> owners = new ArrayList<String>();
     List<String> producers = new ArrayList<String>();
-    
+    Map<String, String> labels = new HashMap<String, String>();
+
+    if (rtoken.getLabelsSize() > 0) {
+      labels = rtoken.getLabels();
+      if (!labels.isEmpty()) {
+        for (Map.Entry<String, String> entry : labels.entrySet())
+        {
```

Comment:
this is not the style used in Warp 10, put the curly brace on the same line as the for/switch statements.